site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -3,3 +3,61 @@ api: runtime
 ---
 
 <!-- Intentionally blank -->
+
+## Examples
+
+Using a content script to inject an extension asset on a page.
```

Comment:
Maybe add a heading for each of the three examples.

```
### Use `getURL` to find an image inside the extension

### Use runtime messaging to communicate between ...

### Retrieve arbitrary data from an extension's manifest
```

Reply:
TBH I've been on the fence about adding headers to these sections. I think you've pushed me over the edge.